Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -8060,7 +8060,9 @@ - (void)_updateInputContextAfterBlurringAndRefocusingElement
         [_asyncSystemInputDelegate invalidateTextEntryContext];
     else
 #endif
+IGNORE_WARNINGS_BEGIN("misleading-indentation")
```

Comment:
This is quite annoying, as it makes the code even harder to read. Perhaps we should just be less smart with `#if`s here.

Reply:
Yep, I presented an alternative in the other source file using braces:
<https://github.com/WebKit/WebKit/pull/22638#discussion_r1449082159>

Or we could just duplicate that one line of code.

I don't really care which one we do—someone just needs to make a decision. :)

Reply:
Seems like there could also just be an early return in the `if (self.shouldUseAsyncInteractions)`?

I will give the final say to @whsieh 🙂

Reply:
I don't have strong preferences either way — but if an early return here is sufficient to avoid suppressing the warning, it might be cleaner to write it that way.